### PR TITLE
ES6/Modules

### PR DESCRIPTION
**Pull request by CakeInTech**
## changes made by @MohamedCK  😄 
- Re-created your Awesome books app following the rules:
    - Used ES6 syntax.
    - Divided your code into modules. Save each module in a separate file in the modules 
      dir. Import modules in the index.js file. For this exercise - the more modules the 
      better!
    - Kept all functionalities of the app without errors.
    - Refactored the methods and functions to arrow functions.
    - Used let and const in the correct way.
- Displayed the current date below the navigation bar.

## Review Changes.

- Used arrow function in ui.js  [here](https://github.com/MohamedCK/Modules-Awesome-Books/blob/Modules/Modules/ui.js#L18-L35)
- Used arrow function in store.js [here](https://github.com/MohamedCK/Modules-Awesome-Books/blob/Modules/Modules/store.js#L12-L26)

### CODE REVIEW

File: Modules/store.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    return books;
+  }
+
+  static addBook(book) {
+    const books = this.getbooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(title) {
```
Comment (OLIPLICHE) [inline Modules/store.js]: - [x] Please use `arrow` syntax for functions. ( 👉 ES6 syntax, Project requirements).

File: Modules/ui.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    });
+  }
+
+  static deleteBook(target) {
+    if (target.classList.contains('remove')) {
+      target.parentElement.parentElement.remove();
+    }
+  }
+
+  static clearFields() {
```
Comment (OLIPLICHE) [inline Modules/ui.js]: - [x]  Please use `arrow` syntax for functions. ( 👉 ES6 syntax, Project requirements).

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @MohamedCK , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- The app is working - it is possible to add and remove books from the list.👏
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Modules/ui.js (hunk 2 above)
Comment (MohamedCK) [reply Modules/ui.js:31]: @OLIPLICHE  

Thank you for the reminder. 😺

**Comment by CakeInTech** [comment]
@OLIPLICHE  

Thank you for the reminder. 😺

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @MohamedCK👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**